src/Spryker/Glue/AppMerchantBackendApi/Mapper/GlueRequestMerchantAppOnboardingMapper.php

```diff
@@ -18,9 +18,9 @@ public function mapGlueRequestTransferToMerchantAppOnboardingRequestTransfer(
     ): MerchantAppOnboardingRequestTransfer {
         $merchantTransfer = new MerchantTransfer();
         $merchantTransfer->fromArray($glueRequestTransfer->getAttributes(), true);
+        $merchantTransfer->fromArray($glueRequestTransfer->getAttributes()['merchant'] ?? [], true);
```

Comment:
are both lines needed? As for me only only 2nd looks valid.

Reply:
I don't know who else sends the request to this endpoint. If it's only from onboarding page it can be removed